test/image/compare_pixels_test.js

```diff
@@ -1,109 +1,154 @@
 var fs = require('fs');
-var path = require('path');
 
-var constants = require('../../tasks/util/constants');
-var getOptions = require('../../tasks/util/get_image_request_options');
+var getMockList = require('./assets/get_mock_list');
+var getRequestOpts = require('./assets/get_image_request_options');
+var getImagePaths = require('./assets/get_image_paths');
 
 // packages inside the image server docker
-var request = require('request');
 var test = require('tape');
+var request = require('request');
 var gm = require('gm');
 
-var TOLERANCE = 1e-6;    // pixel comparison tolerance
-var BASE_TIMEOUT = 500;  // base timeout time
-var BATCH_SIZE = 5;      // size of each test 'batch'
-var running = 0;         // number of tests currently running
+// pixel comparison tolerance
+var TOLERANCE = 1e-6;
+
+// wait time between each test batch
+var BATCH_WAIT = 500;
+
+// number of tests in each test batch
+var BATCH_SIZE = 5;
+
+// wait time between each test in test queue
+var QUEUE_WAIT = 10;
```

Comment:
I'm thinking about making changing these to:

```js
var QUEUE_WAIT = process.env.PLOTLYJS_IMAGE_TEST_QUEUE_WAIT || 10;
```

to better handle machine-to-machine difference